core/java/iesi-core/src/main/java/io/metadew/iesi/openapi/ConnectionParser.java

```diff
@@ -0,0 +1,77 @@
+package io.metadew.iesi.openapi;
+
+import io.metadew.iesi.metadata.definition.connection.Connection;
+import io.metadew.iesi.metadata.definition.connection.ConnectionParameter;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.servers.Server;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ConnectionParser {
+    private static ConnectionParser INSTANCE;
+    public static String[] fakeEnvs = new String[] {"TST", "PRD", "ENVV"};
+
+
+    private ConnectionParser() {}
+
+    public synchronized static ConnectionParser getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new ConnectionParser();
+        }
+        return INSTANCE;
+    }
+
+    public  List<Connection> parse(OpenAPI openAPI) {
+        String name = openAPI.getInfo().getTitle();
+        String description = openAPI.getInfo().getDescription();
+        List<URL> adresses = getAdresses(openAPI.getServers());
+        return adresses.stream().map(address -> {
+            String env = fakeEnvs[adresses.indexOf(address)];
```

Comment:
What if I have more servers/adresses than fake environments?

Reply:
have you looked at this?

Reply:
I think it's better to let the user choose the environment from the UI as you said lastly. I can remove the env variable

Reply:
Okay, but we can discuss the flow of user intervention later.
For now just create a random environment like; 'env1', 'env2', 'env3'....

Reply:
Okay okay